app/code/Magento/Sitemap/Model/Sitemap.php

```diff
@@ -661,7 +673,11 @@ protected function _getMediaUrl($url)
      */
     protected function _getFormattedLastmodDate($date)
```

Comment:
From what I can see in the discussion in #9151 the talked about solution would be to show `The lastmod value in the xml should contain the created_at timestamp` was this changed at some point to show `0000-01-01 00:00:00` instead?

Reply:
Hi @dmanners 
Magento should not be obliged to respond to incorrectly filled database fields through third-party code. But in the formation of sitemap we must take into account the case when the lastmod has zero value.
In the case of the #9151, the field "created_at" may also contain a zero value, and the issue remains.
So I decided that is the better solution.

Reply:
Thank you for the update that has clarified it nicely.